site/index.md

```diff
@@ -79,4 +79,15 @@ Read our [introduction article to Vega-Lite v2 on Medium](https://medium.com/@uw
 
 ## Team
```

Comment:
What do you think about removing this and instead writing: 

Vega-Lite was started at the [Interactive Data Lab at the University of Washington] and now being developed by a [community of contributors](https://github.com/vega/vega-lite/graphs/contributors).

Reply:
I like the sentence better ,but probably still nice to list major contributors.

Reply:
We still list them on the readme and at the link.